custom_components/auth_header/__init__.py

```diff
@@ -28,9 +28,8 @@
     {
         DOMAIN: vol.Schema(
             {
-                vol.Optional(
-                    "username_header", default="X-Forwarded-Preferred-Username"
-                ): cv.string,
+                vol.Optional("username_header", default="X-Forwarded-Preferred-Username"): cv.string,
+                vol.Optional("allow_bypass_login", default=False): cv.boolean,
```

Comment:
To keep the behaviour the same as is I think this value should default to True

Reply:
I agree, that's done